sbin/release.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+
+import utils
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "level",
+    metavar="LEVEL",
+    nargs="?",
+    default="prerelease",
+    choices=[
+        "major",
+        "minor",
+        "patch",
+        "premajor",
+        "preminor",
+        "prepatch",
+        "prerelease",
+    ],
+)
+args = parser.parse_args()
+
+with open("package.json", "r") as f:
+    data = json.load(f)
+    current_version = data["version"]
+    next_version = utils.shell(
+        f"yarn semver {current_version} --increment {args.level} --preid alpha"
+    )
+
+    # Generate new commit
+    print (" - Update versions")
+    utils.shell(f'yarn workspaces foreach version --deferred "{next_version}"')
+    utils.shell(f"yarn version apply --all")
+
+    print(" - Update changelog")
+    utils.shell(f'./sbin/changelog.py full --release "v{next_version}" > CHANGELOG.md')
+
+    print(" - Create release commit and tag")
+    utils.shell(f"git add -u")
+    utils.shell(f'git commit -m "v{next_version}"')
+    utils.shell(f'git tag -a "v{next_version}" -m "v{next_version}"')
```

Comment:
This really needed to be python too? I would argue that at this point, there is no benefit :stuck_out_tongue:

Reply:
Yeah agreed, I've removed `shell=True` now at least since that was quite silly.